js/blocks/controls/repeater.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependendies
+ */
+const { BaseControl } = wp.components;
+
+/**
+ * Internal dependendies
+ */
+import { getRenderedFields } from '../loader/edit';
+
+const BlockLabRepeaterControl = ( props, field, block ) => {
+	const subFields = field.sub_fields ? getRenderedFields( field.sub_fields, props, block ) : null;
```

Comment:
If it's alright, I changed `sub-fields` to `sub_fields`, to enable accessing it like `field.sub_fields`.